### PR TITLE
GEODE-8738: Updated documentation about WAN deployments with sites wi…

### PR DESCRIPTION
**Pull request by albertogpz**
…th the same IP address and port for all gateway receivers

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by davebarnes97** [comment]
@albertogpz Thank you for your contribution.
I have edited your text for format and style, but I am unable to push my changes to your fork. I attach the file here so you can incorporate my changes from your side. Please let me know if you'd like to receive these changes in another way.
Best regards, and thanks!
[GEODE-8738-edit.zip](https://github.com/apache/geode/files/5576363/GEODE-8738-edit.zip)

**Review by davebarnes97 — CHANGES_REQUESTED** [review]
Good material. My review applies to the docs only -- I did not test the code.
Changes submitted in another comment.
Thank you.

**Comment by albertogpz** [comment]
> @albertogpz Thank you for your contribution.
> I have edited your text for format and style, but I am unable to push my changes to your fork. I attach the file here so you can incorporate my changes from your side. Please let me know if you'd like to receive these changes in another way.
> Best regards, and thanks!
> [GEODE-8738-edit.zip](https://github.com/apache/geode/files/5576363/GEODE-8738-edit.zip)

@davebarnes97 , thanks for the review and the appropriate corrections. I agree to all of them.
Might be good that @boglesby or @bschuchardt take a look at the text too.

File: geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb [hunk 1]
```diff
@@ -387,3 +387,51 @@ boglesbymac(ny-3:88715)<v4>:36808 | 5371 | 5            |["boglesbymac(ln-1:8865
...
+
+In order to configure a WAN deployment that hides the gateway receivers behind the same IP address and port, all the gateway receivers must have the same value for the hostname-for-senders parameter (the hostname or IP address to be used by clients to access them) and the same value in the start-port and end-port parameters (the port to be used by clients to access them).
+
+As an example, a deployment in which all gateway receivers of a site are accessed via the "gateway1.mycompany.com" hostname and port "1971", every gateway receiver in that site must be configured as follows:
+
+``` pre
+gfsh> create gateway-receiver --hostname-for-senders="gateway1.mycompany.com" --start-port=1971 --end-port=1971
+```
+
+The following output shows how the receiver side would look like after this configuration if 4 gateway recievers were configured:
```
Comment (alb3rtobr) [inline geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb:404]: There is a typo: "recievers"

**Review by alb3rtobr — CHANGES_REQUESTED** [review]
LGTM, just fix the typo

File: geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb (hunk 1 above)
Comment (albertogpz) [reply geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb]: Good catch, Alberto! Thanks.

**Review by alb3rtobr — APPROVED** [review]

**Review by davebarnes97 — APPROVED** [review]

**Pull request merged**